### PR TITLE
[IMP] user_story: Grouping by User Story State for Acceptability Criteria - task4928

### PR DESCRIPTION
**Pull request by gomezvillegasdaniel**
DUMMY: https://github.com/Vauxoo/instance/pull/237
- [x] Grouping by User Story State for Acceptability Criteria
- [x] Make involved english to spanish translations

### CODE REVIEW

**Comment by zaoral** [comment]
Hi @gomezvillegasdaniel 

Please can you fix [this](https://travis-ci.org/Vauxoo/addons-vauxoo/jobs/115220185#L495-L498) sub groups of flake8 errors
**IMPORTANT: Please omit the odoo lint errores that are after this [line](https://travis-ci.org/Vauxoo/addons-vauxoo/jobs/115220185#L501)**

**Comment by gomezvillegasdaniel** [comment]
@zaoral It's done, thanks. I'm gonna make the dummy again.

**Comment by gomezvillegasdaniel** [comment]
@zaoral Hi. It's done, thanks. Dummy it's rebuilding.

**Comment by gomezvillegasdaniel** [comment]
@JulioSerna

**Comment by JulioSerna** [comment]
:+1:

**Comment by gomezvillegasdaniel** [comment]
@moylop260 Hi, may you merge this, please?

File: user_story/model/user_story.py [hunk 1]
```diff
@@ -616,6 +618,14 @@ def _get_user_story_field(self, cr, uid, ids, fieldname, arg,
             res[ac_brw.id] = copy_field
         return res
 
+    def _get_user_story_state(self, cr, uid, ids, fieldname, arg,
+                              context=None):
+        context = context or {}
```
Comment (moylop260) [inline user_story/model/user_story.py]: Use if context is None

**Comment by moylop260** [comment]
Small comment after that :+1:

**Comment by gomezvillegasdaniel** [comment]
@moylop260 Done

**Comment by moylop260** [comment]
What is that comment?

**Comment by gomezvillegasdaniel** [comment]
:0 I thought you said me to add that comment

**Comment by gomezvillegasdaniel** [comment]
@moylop260 Done

File: user_story/model/user_story.py [hunk 2]
```diff
@@ -616,6 +618,14 @@ def _get_user_story_field(self, cr, uid, ids, fieldname, arg,
             res[ac_brw.id] = copy_field
         return res
 
+    def _get_user_story_state(self, cr, uid, ids, fieldname, arg,
+                              context=None):
+        context = {} if context is None else context
```
Comment (moylop260) [inline user_story/model/user_story.py]: @JulioSerna 
Could you help to @gomezvillegasdaniel in this change, please?

Comment (moylop260) [inline user_story/model/user_story.py]: I'm talking about:
https://doc.odoo.com/contribute/15_guidelines/coding_guidelines_framework.html#the-infamous-context

Comment (moylop260) [inline user_story/model/user_story.py]: Indeed, I saw that you aren't use the `context` then... you can delete this line.

**Comment by gomezvillegasdaniel** [comment]
@moylop260 Done, again :p

**Comment by gquilarque** [comment]
Hi @nhomar 

Could you update this PR en vauxoo production ignoring the milestone?

**Pull request merged**